app/controllers/schools/assignments_controller.rb

```diff
@@ -0,0 +1,41 @@
+module Schools
+  class AssignmentsController < ApplicationController
+    # /school/courses/:course_id/assignments/:id
+    def update
+      @milestones =
+        current_school
+          .courses
+          .find_by(id: params[:course_id])
+          .targets
+          .milestones
+          .order(milestone_number: :asc)
+
+      authorize(@milestones, policy_class: Schools::AssignmentPolicy)
```

Comment:
This looks incorrect you can't call authorize on a collection; it should be a record